docs/api-setup.md

```diff
@@ -0,0 +1,68 @@
+---
+id: api-setup
+title: API Setup
```

Comment:
Consider renaming this page -- "API Setup" makes sense to someone that has the single goal of connecting the API to Postman or Paw. However to someone browsing the docs for the first time, the title of this page could be misleading: "API Setup" gives me the idea of a required step to "turn on" the API, when in reality this page is more specific to API *tooling*

I'm not sure of the best name, and I don't think "tooling" alone will make sense to a broad audience, but food for thought.

Reply:
Good point! Maybe "Setup Postman/Paw for API Testing"?